codecov.yml

```diff
@@ -1,10 +1,5 @@
 # See documentation at https://docs.codecov.io/docs/codecovyml-reference
 
-codecov:
-  notify:
-    # Wait for Jest & PHPUnit $ Behat reports before notifying
-    after_n_builds: 3
-
```

Comment:
Why is this no longer needed?

Reply:
Codecov waits for all 3 coverage reports before posting the summary for the PR; problem is that we are now skipping some PR checks which means it's no longer guaranteed 3 coverage reports will be uploaded, so Codecov will be waiting for reports that it will potentially never receive.